snmp/datadog_checks/snmp/data/default_profiles/nasuni-filer.yaml

```diff
@@ -0,0 +1,334 @@
+      - OID: 1.3.6.1.4.1.42040.2.2.1.24
+        name: volumeTableNumFtpdirs
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.2
+          name: volumeTableDescription
+        tag: volume_table_description
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.3
+          name: volumeTableProvider
+        tag: volume_table_provider
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.4
+          name: volumeTableProtocol
+        tag: volume_table_protocol
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.12
+          name: volumeTableIsActive
+        tag: volume_table_is_active
+        mapping:
+          0: 'false'
+          1: 'true'
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.13
+          name: volumeTableIsShared
+        tag: volume_table_is_shared
+        mapping:
+          0: 'false'
+          1: 'true'
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.14
+          name: volumeTableIsReadOnly
+        tag: volume_table_is_read_only
+        mapping:
+          0: 'false'
+          1: 'true'
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.15
+          name: volumeTableIsPinned
+        tag: volume_table_is_pinned
+        mapping:
+          0: 'false'
+          1: 'true'
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.16
+          name: volumeTableIsRemote
+        tag: volume_table_is_remote
+        mapping:
+          0: 'false'
+          1: 'true'
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.17
+          name: volumeTableAvEnabled
+        tag: volume_table_av_enabled
+        mapping:
+          0: 'false'
+          1: 'true'
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.18
+          name: volumeTableRemoteAccessEnabled
+        tag: volume_table_remote_access_enabled
+        mapping:
+          0: 'false'
+          1: 'true'
+      - column:
+          OID: 1.3.6.1.4.1.42040.2.2.1.5
+          name: volumeTableStatus
+        tag: volume_table_status
+metric_tags:
+  - OID: 1.3.6.1.4.1.42040.1.1.0
+    symbol: filerIdentifier
+    tag: filer_identifier
+  - OID: 1.3.6.1.4.1.42040.1.2.0
+    symbol: filerVersion
+    tag: filer_version
+  - OID: 1.3.6.1.4.1.42040.1.3.0
+    symbol: filerSerialNumber
+    tag: filer_serial_number
+  - OID: 1.3.6.1.4.1.42040.1.101.1.0
+    symbol: filerPlatformName
+    tag: filer_platform_name
+  - OID: 1.3.6.1.4.1.42040.1.101.2.0
+    symbol: filerPlatformType
+    tag: filer_platform_type
+  - OID: 1.3.6.1.4.1.42040.1.101.3.0
+    symbol: filerPackageFormat
+    tag: filer_package_format
+  - OID: 1.3.6.1.4.1.42040.1.101.4.0
+    symbol: filerBiosVersion
+    tag: filer_bios_version
+  - OID: 1.3.6.1.4.1.42040.1.101.5.0
+    symbol: filerCpuModel
+    tag: filer_cpu_model
+  - OID: 1.3.6.1.4.1.42040.1.101.6.0
+    symbol: filerPhysCpuCount
+    tag: filer_phys_cpu_count
+  - OID: 1.3.6.1.4.1.42040.1.101.7.0
+    symbol: filerCoreCount
+    tag: filer_core_count
```

Comment:
I think those should be scalar metrics instead.

Reply:
Is it worth emitting it as a metric ? They're not supposed to change I believe
Maybe I can just remove these tags

Reply:
There are cases where a value that doesn't might still be useful as metric, for example we have often metrics like "total memory" (similar filerCoreCount, it's not supposed to change, but it can still be useful to have the value in DD).

Reply:
Makes sense, added them as scalar metrics 👍